src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
```

Comment:
- [ ] kindly remove the `/* eslint-disable */` commanded and try to fix the linter errors to keep your code readable and clean 🙏

Reply:
Thank you, I have removed that comment 👍 💯 !